projects/igniteui-angular/src/lib/grids/common/crud.service.ts

```diff
@@ -97,10 +97,30 @@ export class IgxCell {
         public rowIndex: number,
         public column,
         public value: any,
-        public editValue: any,
+        public _editValue: any,
         public rowData: any,
         public grid: GridType) {
         this.grid.validation.create(id.rowID, rowData);
+        this.editValue = this._editValue;
+    }
+
+    public get editValue() {
+        const formControl = this.grid.validation.getFormControl(this.id.rowID, this.column.field);
+        if (formControl) {
+            return formControl.value;
+        }
```

Comment:
Kind of surprised, this doesn't break the model bind with blur if the value changes, yet the getter still returns the old one. It works, but still perhaps it'd be better if we return the pending value for blur for consistency?